src/main/scala/org/scalacheck/magnolia/arbitrary.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2018 com.github.chocpanda
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://app.codacy.com/manual/ChocPanda/scalacheck-magnolia/pullRequest?prid=4177247)